exporter/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM gcr.io/google-appengine/debian9
+
+
+# Get the list of packages from the base image.
+ENV EXPORTER_VERSION 1.3.1
+ENV EXPORTER_SHA1 d4459037d64b07a6ca7d316904e246b729930a69822301d0c7a018913c9c1fd1
```

Comment:
this env is not used

Reply:
Removed.

Reply:
As now I am building the exporter I can't checksum it.